Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
+
+# Specify your gem's dependencies in cfme-versions.gemspec
+gemspec
```

Comment:
I pretty much specifically designed this package so you didn't need to use bundler to work with it.  Both `minitest` and `rake` are packaged with every modern version of ruby, so this serves no purpose.

Reply:
the Gemfile is required if anyone clones the repo and does bundle install otherwise you get "Could not locate Gemfile".  Since Travis does that by default, it's required for Travis.

Reply:
Fair enough.  That said, you shouldn't need to `bundle install` anything to build this gem, as the dependencies are baked into ruby installations at this point.

And for `Travis`, you can configure this to do nothing for the `install` step:

https://docs.travis-ci.com/user/languages/ruby/#bundler

FYI.